latex_engine_cmds.bzl

```diff
@@ -201,71 +35,80 @@ def add_flags(ctx):
                 "Value of attr format ({}), ".format(ctx.attr.format) +
                 "conflicts with value of flag {}".format(value),
             )
-        flags.append(prefix + value)
+        flags.append(value)
+    dir_count = len(ctx.outputs.out.path.split("/")) - 1
+    dir_pos = dir_count * "../"
+    flags.append("--bibtex-cmd=" + dir_pos + bib_tool.path)
+    flags.append("--bibtex-args=" + "--input-directory=" + dir_pos)
+
+    # TODO: Add debug option
+    # flags.append("--verbose-cmds")
```

Comment:
You could use [config_setting](https://bazel.build/reference/be/general#config_setting) with `compilation_mode` for this

Reply:
answered in the last comment